README.md

```diff
@@ -34,9 +34,9 @@ Features:
 * `desired_capacity` - The desired number of containers running in the service. Default is `1`.
 * `max_capacity` - The maximum number of containers running in the service. Default is same as `desired_capacity`.
 * `min_capacity` - The minimum number of containers running in the service. Default is same as `desired_capacity`.
-* `scaling_metric` - A type of target scaling. Needs to be either `cpu` or `memory`. Default is `null`.
-* `scaling_threshold` - The percentage in which the scaling metric will trigger a scaling event. Default is `null`.
-* `efs_config` - The EFS id, root directory, and path. The module currently supports only one mount. Default is `null`.
+* `scaling_metric` - A type of target scaling. Needs to be either `cpu` or `memory`.
+* `scaling_threshold` - The percentage in which the scaling metric will trigger a scaling event.
+* `efs_config` - The EFS id, root directory, and path. The module currently supports only one mount.
```

Comment:
Add the default value into the description. Each variable has this even if it is an empty value.

Reply:
I feel like if it's an optional variable and the default is "do nothing at all", then it doesn't really make sense to say what the "default value" is.